templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -444,20 +444,28 @@ contents:
       # Then activate all the connections
       for conn in "${connections[@]}"; do
         local active_state=$(nmcli -g GENERAL.STATE conn show "$conn")
-        if [ "$active_state" != "activated" ]; then
-          for i in {1..10}; do
-            echo "Attempt $i to bring up connection $conn"
-            nmcli conn up "$conn" && s=0 && break || s=$?
-            sleep 5
-          done
+        # Connection already active, only set autoconnect
+        if [ "$active_state" == "activated" ]; then
+          nmcli c mod "$conn" connection.autoconnect yes
+          continue
+        fi
+        # Connection inactive, bring it up and set autoconnect
+        for i in {1..10}; do
+          echo "Attempt $i to bring up connection $conn"
+          o="$(nmcli conn up "$conn" 2>&1)" && s=0 || s=$?
           if [ $s -eq 0 ]; then
             echo "Brought up connection $conn successfully"
-          else
-            echo "ERROR: Cannot bring up connection $conn after $i attempts"
-            return $s
+            break
+          elif [[ $o =~ "No suitable device found for this connection" ]]; then
+            echo "WARNING: Skipping activation of $conn: $o"
```

Comment:
I would be interested in adding a comment here on why this might be happening so when we look at it after we wonder about why we added it.
This is tied to a situation where one of the slave clones might not be available but we could still work with the other.
So perhaps the proper fix is to account for this in general, instead of parsing a specific error.
So keep track of the number of slave connections that failed and survive if at least one of them is ok.